### PR TITLE
SLING-9827 Specify the API Region to export packages into

### PR DESCRIPTION
**Pull request by bosschaert**
The region can be specified with the -e / --exports-to-region option.
Normally when this option is used the region specified with this option
would be the 'global' region or a region specific to the content
package.

### CODE REVIEW

File: src/main/java/org/apache/sling/feature/cpconverter/features/DefaultFeaturesManager.java [hunk 1]
```diff
@@ -225,6 +224,45 @@ public void addConfiguration(String runMode, String pid, Dictionary<String, Obje
         }
     }
 
+    private void addAPIRegions(Feature feature, List<String> exportedPackages) {
```
Comment (cziegeler) [inline src/main/java/org/apache/sling/feature/cpconverter/features/DefaultFeaturesManager.java:227]: The regions java api supports generating (and parsing) of the extension JSON - we should not do this by hand anymore

Comment (bosschaert) [reply src/main/java/org/apache/sling/feature/cpconverter/features/DefaultFeaturesManager.java:227]: :) this class was already doing it like that - but I'll be happy to improve it to use the API.

Comment (cziegeler) [reply src/main/java/org/apache/sling/feature/cpconverter/features/DefaultFeaturesManager.java:227]: Thanks, yes, I know but I think if we touch the code we should improve :)

Comment (bosschaert) [reply src/main/java/org/apache/sling/feature/cpconverter/features/DefaultFeaturesManager.java]: You are right - this is better. I've just made the change. Thanks for reviewing @cziegeler !

**Review by cziegeler — APPROVED** [review]
Lgtm, thanks

**Pull request merged**